src/Models/ProductCategoryHasProductsFilter.php

```diff
@@ -5,7 +5,7 @@
 class ProductCategoryHasProductsFilter extends Filter
 {
     public string $typeDefinition = "Relewise.Client.Requests.Filters.ProductCategoryHasProductsFilter, Relewise.Client";
-    public static function create(bool $negated = false) : ProductCategoryHasProductsFilter
+    public static function create(bool $negated) : ProductCategoryHasProductsFilter
```

Comment:
This is due to a breaking change in the .NET SDK. So we are okay with breaking the PHP SDK as well.